hypertrace-graphql-labels-schema/src/main/java/org/hypertrace/graphql/label/request/LabelUpdateRequest.java

```diff
@@ -0,0 +1,10 @@
+package org.hypertrace.graphql.label.request;
+
+import org.hypertrace.core.graphql.context.GraphQlRequestContext;
+import org.hypertrace.graphql.label.schema.Label;
+
+public interface LabelUpdateRequest {
+  GraphQlRequestContext context();
+
+  Label newLabel();
```

Comment:
`newLabel` would mean it's creating a new label

```suggestion
  Label label();
```